scripts/migrate-black.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# check out every commit added by the current branch, blackify them,
+# and generate diffs to reconstruct the original commits, but then
+# blackified
+import os
+import sys
+from subprocess import check_output, run, Popen, PIPE
+
+
+def git(*args):
+    return check_output(["git"] + list(args)).decode("utf8").strip()
```

Comment:
Why are you stripping newlines here? Couldn't that messup output?

Reply:
no, but we need to get rid of the whitespace some versions of git add before/after its output